### ISSUE TITLE
.item() called twice in ppo_actor.py

### ISSUE THREAD

**Issue by aniltrkkn** [OpenRLHF/OpenRLHF#994]
in ppo_actor.py:
```
if self.args.use_kl_loss:

            if self.args.init_kl_coef > 0:
                kl = compute_approx_kl(
                    action_log_probs,
                    base_action_log_probs,
                    kl_estimator=self.args.kl_estimator,
                )
            else:
                kl = torch.zeros_like(action_log_probs, dtype=action_log_probs.dtype, device=action_log_probs.device)
            kl_loss = masked_mean(kl, experience.action_mask)
            experience.info["kl"] = kl_loss.detach().item()
        else:
            kl_loss = 0
```

then we do 

```
        for k, v in experience.info.items():
            status[k] = v.mean().item()
```

This will causes an error in .mean() since first kl_loss.detach().item() converts it to a float.
